standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -13339,5 +13351,6 @@ private void recoverInvalidScheduledQueryState(int timeoutSecs) {
         pm.makePersistent(e);
       }
     }
+    q.closeAll();
```

Comment:
use try-with-resources in this method

Reply:
the method ```recoverInvalidScheduledQueryState``` originally has not a try clause, so use ```q.closeAll()``` directly,  maybe another jira is needed to address the problem?